src/Validation/Validation.php

```diff
@@ -1020,6 +1022,45 @@ public static function uploadedFile($file, array $options = [])
         return true;
     }
 
+/**
+ * Validates a geographic coordinate.
+ *
+ * Supported formats are right now:
+ *
+ * - <latitude>, <longitude> Example: -25.274398, 133.775136
+ *
+ * ### Options
+ *
+ * - `type` - A string of the coordinate format, right now only `longLat`.
```

Comment:
So you want three?

* longitude
* latitude
* geoCoordinate (that does both above if they're combined)